### DISCUSSION TITLE
ProvideTag with mutation ?

### DISCUSSION THREAD

**Discussion by VaZark** [reduxjs/redux-toolkit#3276] [Q&A]
With most POST requests, our project tends to return the updated result. it'd be cool if it were possible to update the result from the query  with the result from a mutation instead of invalidating the query's Tag and forcing a refetch for each change  

Is there a way to assign a tag (or some ID) to each object of a pagination result and replace it with a mutation's result?

**Comment by phryneas** [comment]
No, you would have to do that manually with [`updateCacheData`](https://redux-toolkit.js.org/rtk-query/api/created-api/api-slice-utils#updatequerydata).

A tag is just something you slap onto a cache entry, really think of a post-it or something similar on a closed box. It describes what's in there, but in no way describes the shape of other stuff that might be in there as well.
